test/internal/cmd/generate-gateway-api-consts/main.go

```diff
@@ -56,6 +59,10 @@ func flagParse() {
 		log.Print("Please provide the 'gateway-api-version' flag")
 		os.Exit(0)
 	}
+	if *gatewayAPIPackageVersionFlag == "" {
+		log.Print("Please provide the 'gateway-api-package-version' flag")
+		os.Exit(0)
+	}
```

Comment:
Does this indeed require it always, without a default fallback version managed by renovate or dependabot? This looks like it would block runs not initiated from the make targets, which I use if I want to run a debuggered test in a way the make targets don't support.

Reply:
I did not consider the possibility of it being called without using make.

I considered setting `gateway-api-package-version` to the same value as `gateway-api-version` when it is not specified.

OR

Automatically retrieve its version from go mod?

WDYT?

Reply:
Either of those seems fine. I'm less familiar with how we're using `gateway-api-version` (and it looks like it has no default?) but if it is getting set automatically that's fine. I'd say go with whichever is easier.